.github/workflows/style.yml

```diff
@@ -0,0 +1,82 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    paths:
+      [
+        "**.[rR]",
+        "**.[qrR]md",
+        "**.[rR]markdown",
+        "**.[rR]nw",
+        "**.[rR]profile",
+      ]
+    workflow_dispatch:
```

Comment:
question: what does this do at this indentation level?  should it be on level to the left?

question: I thought you found styling & documenting interfered with each other sometimes, and chose to make like... documentation pushes automatic, and styling into just a check + maybe a workflow_dispatch command to actually do the styling & push?

Reply:
suggest: if you've set up workflow_dispatch commands for this, then please document them somewhere... maybe the PR template?

Reply:
- Ah yes, good catch, the indentation is off.
- It's less that styling and documentation interfere with each other and more that once the document action pushes, the pushed commit doesn't rerun CI. So workflow_dispatch allows us to retrigger without another push.
- Can do that.

Reply:
Oh, that's what workflow_dispatch means?  I mean, that's what is sounds like, but not what skimming a few lines of docs made me think / what I thought earlier plan was; I thought these were special slash commands for comments.

Reply:
Ah yes, it lets you trigger the workflow from the Actions tab at the top, like [here](https://github.com/cmu-delphi/epiprocess/actions/workflows/pkgdown.yaml).